src/git-command-manager.ts

```diff
@@ -338,13 +319,9 @@ class GitCommandManager {
     }
 
     // Minimum git version
```

Comment:
This is just reaction because i needed to export `MinimumGitVersion`. The previous comments about `minimumGitVersion` moved to the new field that is exported (top of this file)